src/templates/index.html

```diff
@@ -5,7 +5,7 @@
 <html>
 
         <title><%=settings.title%></title>
-    <script>
+    <script nonce="<%=settings.nonce && settings.nonce.padLicense || ''%>">
```

Comment:
A nonce should only be used once: refreshing should get a different nonce, and two different users should never see the same nonce. If the same nonce is used every time, an attacker could visit the page, look at the nonce, then use it in the injected code.